FusionIIIT/templates/financeAndAccountsModule/hello.html

```diff
@@ -0,0 +1 @@
+Print(hello)
```

Comment:
Don't push the unnecessary file. Also, keep the commit message concise and clear.